draft/examples/valid/highered-course.json

```diff
@@ -36,10 +37,17 @@
     "Operation Systems",
     "Computer Structures"
   ],
+  "isAccessibleForFree": true,
+  "conditionsOfAccess": {
+    "id": "http://w3id.org/kim/conditionsOfAccess/no_login",
+    "type": "Concept"
```

Comment:
Warum ist hier `type` mit angegeben, bei `educationalLevel` aber nicht?

Ist beides korrekt, da `type` bei beiden Attributen angegeben werden KANN.
(trifft auch auch `highered-course-part.json` zu.

Frage mich, ob das eher verwirrt, wenn in den Beispielen mal so oder so mit optionalen Werten umgegangen wird oder die Beispiele möglichst vollständig sein sollten

Reply:
Ich würde das auch ohne `type` machen. Habe das auch im OERSI ohne `type` umgesetzt.

Reply:
In der genannten Datei steht `"type": [ "Concept" ]` auch bei `about` und `audience`.

Da es im Schema explizit genannt wird, aber nicht `required` ist, finde ich es schon sinnvoll, wenn es in einigen – aber eben nicht allen – Beispielen auftaucht. Wenn, dann sollten wir das auch komplett aus dem Schema tilgen, womit ich d'accord wäre. Die genannten Informationen finden sich ja ohnehin im RDF, wenn die URI dereferenziert wird. Ich schlage vor, das hier erstmal zu mergen und die Frage in einem gesonderten Ticket zu diskutieren.